plugins/ruby-foreman-tasks/debian/changelog

```diff
@@ -1,3 +1,15 @@
+ruby-foreman-tasks (8.3.3-1) stable; urgency=low
+
+  * 8.3.3 released
+
+ -- Adam Ruzicka <aruzicka@redhat.com>  Thu, 23 Nov 2023 16:39:46 +0100
+
+ruby-foreman-tasks (8.3.3-1) stable; urgency=low
```

Comment:
Somehow this was log entry was duplicated

Reply:
Sorry about that, fixed